SU-HSDO/suhumsci
# shs-55: edit breakout box wysiwyg class and color

## Summary
This changes the 'breakout box' WYSIWYG style with a new class `hs-breakout-box` and changes the color from the previous setting for well to `spotlight`. 

## Steps to Test
1. Import the config and create a new flexible page.
2. Add a text component, add some text, highlight it and add the 'Breakout Box' style (screenshot below).
3. You should be able to add this class and have it applied for all text formats _except_ Minimal HTML.
4. View the page and verify it's using the theme's 'spotlight color'
5. Verify the above works for the Traditional theme as well. 

## Screenshots
![Screen Shot 2022-09-19 at 9 46 49 AM](https://user-images.githubusercontent.com/8405274/191033470-b399a96d-b60b-4e3a-8619-7f569ed6c64d.png)

![Screen Shot 2022-09-19 at 9 47 02 AM](https://user-images.githubusercontent.com/8405274/191033493-06cfb310-9375-43aa-bd7f-f3ed9d8e4a47.png)
